compiler/rustc_middle/src/mir/syntax.rs

```diff
@@ -671,6 +671,9 @@ pub enum TerminatorKind<'tcx> {
         /// This allows the memory occupied by "by-value" arguments to be
         /// reused across function calls without duplicating the contents.
         args: Vec<Operand<'tcx>>,
```

Comment:
Could also turn this into a `Vec<Spanned<Operand<'tcx>>>`

Reply:
I tried, but to do that we need to make Spanned derive TypeFoldable and TypeVisitable (which TerminatorKind already does), and to do that we need to make rustc_span depend on rustc_middle, and I don't think we want that?

Reply:
You should be able to add a manual impl to rustc_middle. We already have a bunch of them there. Or ad a generic impl to rustc_span in case that already depends on rustc_type_ir

Reply:
Worst case make it a `Vec<(Span, Operand)>`

Reply:
Oh. Right. Done! (`Spanned`.)